desk/app/docket.hoon

```diff
@@ -389,6 +389,8 @@
       ?.  (~(has by charges) desk)
         `state
       =/  =charge  (~(got by charges) desk)
+      ?:  &(?=(%install -.chad.charge) ?=(%held zest))
```

Comment:
@philipcmonk just want to double check that seems ok. we found that new apps were showing suspended first which was kinda weird UX.

Reply:
bump, @philipcmonk

Reply:
@philipcmonk ?

Reply:
This will make them not show up at all until they're running, right?  That seems fine, though if you wanted to show an explicit "installing" state, I think you could do that on the frontend by checking this same condition.

(if I'm not seeing a gh notification, feel free to ping me out of band)